cmd/gluetun/main.go

```diff
@@ -320,6 +320,9 @@ func _main(ctx context.Context, buildInfo models.BuildInformation,
 	if err := routingConf.SetOutboundRoutes(allSettings.Firewall.OutboundSubnets); err != nil {
 		return err
 	}
+	if err := routingConf.AddLocalRules(localNetworks); err != nil {
+		return err
+	}
```

Comment:
nit (I know my code doesn't follow this, I need to update it):

- Avoid using the short if form it makes it harder to read especially as line length increases (and then it becomes debatable what is the max length)
- Wrap the error to have contextual information on what failed and to help debug

```suggestion
	err = routingConf.AddLocalRules(localNetworks)
	if err != nil {
		return fmt.Errorf("adding local rules: %w", err)
	}
```